pkg/agent/flowexporter/connections/connections.go

```diff
@@ -26,6 +27,14 @@ import (
 	"antrea.io/antrea/pkg/agent/proxy"
 )
 
+const (
+	periodicDeleteInterval = time.Minute
+)
+
+var (
+	StaleConnectionTimeout = 5 * time.Minute
```

Comment:
it may be a better pattern to have `const DefaultStaleConnectionTimeout = 5 * time.Minute` and pass the actual value as a parameter to the connection store when instantiating it, but I'll let you make that call

Reply:
Following the same pattern as active and idle timeouts make sense. I changed it. PTAL.